README.md

```diff
@@ -22,7 +22,7 @@ This application is available under license from Oppkey, Inc.
 
 ## Pre-requisites
 
-* Flutter, dev channel
+* Flutter, master channel
```

Comment:
This is likely only a temporary change.  Suggest you add a date and note.  We intend to use the dev channel after the next update.  It looks like the Flutter dev channel was updated 13 days ago.  Please add a note that we should test the dev channel again when it is updated.

![image](https://user-images.githubusercontent.com/5799860/98952188-02844900-24b0-11eb-97c4-4cdbb4584978.png)